agent/tool-scripts/tests/blktrace/gold/stderr

```diff
@@ -2,12 +2,19 @@ blktrace: Error, one of the following options is required, --install|--start|--s
 
 The following options are available:
 
+Operational options - do *not* specify any of these when you register a tool:
+
 	--install                 verify this perf tool will work
 
 	--start|stop|postprocess  start/stop/postprocess the data collection
 	-d str|--dir=str          directory to store data collection (required)
+
+Tool-specific options - these should be specified when you register a tool:
+
 	--devices=str,[str]       the list of block devices to trace (required w/ --start)
 
+Help option - do *not* specify this when you register a tool:
+
 	-h|--help                 display this help message
 blktrace: WARNING - postprocess - tool output directory, '/var/tmp/pbench-test-tool-scripts/blktrace/tools-group/blktrace', missing
```

Comment:
Iff you have to make a change for some other reason, I would request another blank line after the `-h` line...oh, nevermind -- that would require changes to nearly 40 gold files.  😝